docs/integrations/sources/apify-dataset.md

```diff
@@ -1,11 +1,26 @@
+---
+description: >-
+  Web scraping and automation platform.
+---
+
 # Apify dataset
 
 ## Overview
 
+[Apify](https://www.apify.com) is a web scraping and web automation platform providing ready and custom made solutions, [SDK](https://sdk.apify.com/),
+and other tools, such as proxies, to build and run any job at scale. 
+
+The results of a scraping job are usually stored in [Apify Dataset](https://docs.apify.com/storage/dataset). This Airbyte connector allows you
+to automatically sync the contents of a dataset to your chosen destination using Airbyte.
+
+To sync data from a dataset, all you need to know is its ID. You will find it in [Apify console](https://my.apify.com/) under storages.
 
-The Apify dataset connect supports full refresh sync only.
+### Running Airbyte sync from Apify webhook
```

Comment:
Do we need to put together a POST request body payload? If so then let's add an example

Reply:
Adding this screenshot
<img width="2330" alt="CleanShot 2021-08-16 at 11 11 28@2x" src="https://user-images.githubusercontent.com/1821884/129539865-695ebd2f-6a15-49b3-a10e-42d7ebc10bba.png">